content/guides/virtualizing/aws.md

```diff
@@ -0,0 +1,156 @@
+
+### Preparing the virtual disk image <a name="part_prepare">
+
+##### Spotting the virtual disk image
+
+After following through the process, you should now have a `.vdi` file alongside other relevant. Its precise location can vary according to your operating system, but generally speaking;
+
+- If you're using Windows, you can find the files of Haiku's virtual machine in the following directory:
+
+```
+C:\Users\<YOUR USERNAME>\.VirtualBox
+```
+
+Replace `<YOUR USERNAME>` with your username.
+
+- In Linux and Solaris operating systems, you can find them in your home directory:
+
+```
+~/VirtualBox VMs
+```
+
+- In Mac OS X environments, the files can be found here instead:
+
+```
+~/Library/VirtualBox
+```
+
+Since `.vdi` files are not supported by one of the tools that we will use later, we will have to convert our `.vdi` file to a `.vhd` file, which shares a lot of similarities and is also properly supported. In order to do that, we will quickly use a command that VirtualBox provides to us and wait for the process to finish:
+
+```
+vboxmanage clonehd <source path of the .vdh file> <destination path of the .vhd file> --format VHD
+```
+
+### Preparing the environment <a name="part_environment">
+
+Now that we have successfully extracted the `.vdi` file, we will need to convert it into an AMI image. Fortunately for us, Amazon provides tools that are designed for this operation.
+
+- In Debian/Ubuntu machines, you can obtain them by running the following command inside of a Terminal:
+
+```
+sudo apt install ec2-api-tools ec2-ami-tools
+```
+
+- In Linux-based operating systems that use the [Red Hat Package Manager](https://rpm.org) (RPM), such as CentOS, Fedora, Red Hat Linux, you can obtain the tools by running these commands instead:
+
+```
+sudo yum install ruby
+
+# Install ec2-api tools
+wget http://s3.amazonaws.com/ec2-downloads/ec2-api-tools.zip
+unzip ec2-api-tools.zip
+cd ./ec2-api-tools-*
+mkdir /opt/ec2 # feel free to use other directories
+mv ./* /opt/ec2/tools
+
+
+# Install ec2-ami-tools
+wget https://s3.amazonaws.com/ec2-downloads/ec2-ami-tools.noarch.rpm
+sudo yum install ec2-ami-tools.noarch.rpm
+```
+
+Make sure to fill out the information in between of the `<>`'s.
+
+We're nearly done! All we need is to store our credentials in variables, since we will use them multiple times during the next section of the tutorial.
+
+For convenience, we will set our [AWS Security Credentials](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html) as two variables; `AWS_KEY` and `AWS_SEC`.
+
+```
+export AWS_KEY="<YOUR AWS KEY>"
+export AWS_SEC="<YOUR AWS SECRET>"
+```
+
+Once again, make sure to replace the two placeholders in between of the `<>`'s with your credentials.
+
+### Preparing the AMI <a name="part_ami">
+
+##### Uploading the disk image
+
+Now that you have gotten everything set and in order, we can now upload the image over to Amazon!
```

Comment:
In this guide you decided to mostly used "we now do this", "we now do that". We should avoid switching to "you now do this", "you now do that". So:
> Now we got everything set and in order, we upload the image to Amazon!